php-ts-mode.el

```diff
@@ -102,6 +105,10 @@
     "while" "yield")
   "PHP keywords for tree-sitter font-locking.")
 
+(defvar php-ts-mode--built-in-functions
+  '("die" "echo" "empty" "isset")
```

Comment:
Echo is already in `php-ts-mode--keywords` https://github.com/emacs-php/php-ts-mode/blob/ef67457ec31462bf78b615d6b47b64f413843d65/php-ts-mode.el#L98C41-L98C47 

If we add `isset` shouldn't we also have `unset` here ?

Reply:
`echo` was removed from `php-ts-mode--built-in-functions`
`unset` is parsed differenly by treesitter, it is a `unset_statement`, not simple `function_call_expression`, so individual font rule was added

Reply:
I see now. `die`, `empty` and `isset` are represented as:
```
 (expression_statement
  (conditional_expression
   condition: 
    (function_call_expression function: (name)
     arguments: 
      (arguments (
       (argument
        (variable_name $ (name)))
       )))
```
Makes sense.